sklearnex/tests/test_patching.py

```diff
@@ -129,6 +133,8 @@ def test_standard_estimator_patching(caplog, dataframe, queue, dtype, estimator,
             and dtype in [np.uint32, np.uint64]
         ):
             pytest.skip("Windows segmentation fault for Ridge.predict for unsigned ints")
+        elif estimator == "KMeans" and queue and queue.sycl_device.is_gpu:
+            pytest.skip("KMeans does not support GPU queues")
```

Comment:
This may change when out of preview is finalized

Reply:
Definitely correct. I will have to leave a note in the other PR to specifically remove these lines.